src/test/java/driver/singletondriver/ChromeDriverManager.java

```diff
@@ -0,0 +1,49 @@
+package driver.singletondriver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import staticdata.WebTimeouts;
+import utilities.PropertiesManager;
+
+import java.util.concurrent.TimeUnit;
+
+public class ChromeDriverManager {
+
+    private static ChromeDriverManager instanceOfChromeDriver;
+    private WebDriver driver;
+
+    private ChromeDriverManager() {
+        PropertiesManager propertiesManager = new PropertiesManager();
+        System.setProperty("webdriver.chrome.driver", propertiesManager.get("PATH_TO_CHROME_DRIVER"));
+        driver = new ChromeDriver();
+    }
+
+    public static ChromeDriverManager getInstanceOfChromeDriver() {
+        if (instanceOfChromeDriver == null) {
+            instanceOfChromeDriver = new ChromeDriverManager();
+        }
```

Comment:
We don't need implementation of singleton manager for current task, please remove it